archive-recipe.php

```diff
@@ -12,14 +12,14 @@
         <div class="col s-col1 force-col"></div>
         <div class="col s-col23 border-bottom border-left border-right
           ">
-          <h2>Search by Meal:</h2>
+          <h2><?php echo __('[:es]Buscar por ...[:en]Search by ...'); ?></h2>
```

Comment:
This is placeholder. I'm not sure yet what this taxonomy is going to be called so that is just there like that